rktl_planner/CMakeLists.txt

```diff
@@ -3,13 +3,32 @@ project(rktl_planner)
 
 # find catkin so we can use our macros
 find_package(catkin REQUIRED COMPONENTS
-    rktl_msgs
+  rospy
+  rktl_msgs
+  std_srvs
+  std_msgs
+  geometry_msgs
+  message_generation
 )
 
 ## Uncomment this if the package has a setup.py. This macro ensures
 ## modules and global scripts declared therein get installed
 ## See http://ros.org/doc/api/catkin/html/user_guide/setup_dot_py.html
 catkin_python_setup()
 
+# msg and srv
+add_service_files(
+  FILES
+  CreateBezierPath.srv
+)
```

Comment:
We have the `rktl_msgs` package for a reason. I think it makes sense to stick this in there.

Reply:
That is a private service only being used by nodes in the same package. Which is why we didn't add it to the global msgs package.